src/Cardano/Wallet.hs

```diff
@@ -276,9 +280,9 @@ data WalletMetadata = WalletMetadata
     , passphraseInfo
         :: !WalletPassphraseInfo
     , status
```

Comment:
Ah this is the term used in the API spec.

Maybe change the record-field too:
```suggestion
    , state
```

Reply:
Good catch :+1: